intel_extension_for_transformers/llm/runtime/graph/__init__.py

```diff
@@ -48,30 +48,31 @@ def __import_package(self, model_name):
             import intel_extension_for_transformers.llm.runtime.graph.opt_cpp as cpp_model
         elif model_name == "bloom":
             import intel_extension_for_transformers.llm.runtime.graph.bloom_cpp as cpp_model
-        elif model_name == "chatglm2":
+        elif model_name == "chatglm":
```

Comment:
both chatglm and chatgml2?

Reply:
which is fixed in https://github.com/intel/intel-extension-for-transformers/pull/382, I will merge PR382